static/js/pages/candidate-single.js

```diff
@@ -4,11 +4,41 @@
 
 var $ = require('jquery');
 
+var events = require('fec-style/js/events');
+
+var maps = require('../modules/maps');
 var tables = require('../modules/tables');
 var helpers = require('../modules/helpers');
 var columnHelpers = require('../modules/column-helpers');
 var columns = require('../modules/columns');
 
+function buildStateUrl($elm) {
+  return helpers.buildUrl(
+    ['committee', $elm.data('committee-id'), 'schedules', 'schedule_a', 'by_state'],
+    {cycle: $elm.data('cycle'), per_page: 99}
+  );
+}
+
+function highlightRowAndState($map, $table, state, scroll) {
```

Comment:
Is this the same function that's used on the committee state map? Should it be exported from some common place so it's not duplicated?